src/m365/spo/commands/knowledgehub/knowledgehub-remove.spec.ts

```diff
@@ -163,40 +161,20 @@ describe(commands.KNOWLEDGEHUB_REMOVE, () => {
           opts.headers['X-RequestDigest'] &&
           opts.data) {
           if (opts.data === `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><ObjectPath Id="29" ObjectPathId="28"/><Method Name="RemoveKnowledgeHubSite" Id="30" ObjectPathId="28"/></Actions><ObjectPaths><Constructor Id="28" TypeId="{268004ae-ef6b-4e9b-8425-127220d84719}"/></ObjectPaths></Request>`) {
-            return Promise.resolve(JSON.stringify([
+            return JSON.stringify([
               {
                 "SchemaVersion": "15.0.0.0", "LibraryVersion": "16.0.7018.1204", "ErrorInfo": {
                   "ErrorMessage": "An error has occurred", "ErrorValue": null, "TraceCorrelationId": "965d299e-a0c6-4000-8546-cc244881a129", "ErrorCode": -1, "ErrorTypeName": "Microsoft.SharePoint.Client.InvalidClientQueryException"
                 }, "TraceCorrelationId": "965d299e-a0c6-4000-8546-cc244881a129"
               }
-            ]));
+            ]);
           }
         }
       }
 
-      return Promise.reject('Invalid request');
+      throw 'Invalid request';
     });
 
     await assert.rejects(command.action(logger, { options: { debug: true, confirm: true } } as any), new CommandError('An error has occurred'));
   });
-
-  it('correctly handles a random API error', async () => {
```

Comment:
Why did you remove these two tests?

Reply:
The supports checks are done automatically, and @milanholemans told me in an earlier PR that, if I saw those statements, I could remove them. 

The handles a random API error seemed unneeded, as this is a CSOM call and this response is not representative of a realistic response.